### PR TITLE
[stable/elastalert] Allow communication with https ES instances

### PR DESCRIPTION
**Pull request by AlexRRR**
**What this PR does / why we need it**:
Currently it is not possible to communicate with https only endpoints (i.e. AWS Elasticsearch instances) enabling this simple flag in the configuration allows it.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by AlexRRR** [comment]
CLA signed

**Comment by timstoop** [comment]
You also need to bump the minor version in Chart.yaml, all changes in a chart require a version bump and as you're adding a new feature, you can bump the minor.

**Comment by AlexRRR** [comment]
Thanks for the heads up @timstoop, I've bumped the minor version of the chart to `0.1.3`

**Comment by timstoop** [comment]
That's the patch level :) You can bump it to 0.2.0, as you're adding a new feature and can thus bump the minor (MAJOR.MINOR.PATCH).

**Comment by AlexRRR** [comment]
I see! Thanks for the hand-holding @timstoop , really appreciate it

File: stable/elastalert/values.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ elasticsearch:
   host: ""
   # elasticsearch port
   port: 80
+  # use_ssl allows to use ES instances via https (True|False)
+  use_ssl: "False"
```
Comment (unguiculus) [inline stable/elastalert/values.yaml]: Camel case, please.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
cc @pickledrick @jertel

**Comment by jertel** [comment]
Looks ok to me.

**Comment by AlexRRR** [comment]
@unguiculus thanks for reviewing. I never know with 3 letter acronyms how to camelcase. hopefully `useSsl` is the right choice. It seems other charts in stable have that.

**Comment by AlexRRR** [comment]
Closing in favor of #5269

**Pull request closed without merging**